antarest/study/web/output_blueprint.py

```diff
@@ -530,4 +534,37 @@ def get_output_variables_list(uuid: str, output_id: str) -> OutputVariablesList:
         output_id = sanitize_string(output_id)
         return output_service.get_output_variables_list(uuid, output_id)
 
+    @bp.get(
+        "/studies/{uuid}/output/{output_id}/variables-views/data",
+        summary="Fetches the variables view for a given output and a given configuration",
+    )
+    def get_output_variables_view(
+        uuid: str,
+        output_id: str,
+        type: OutputVariablesType,
+        variable_name: str,
+        frequency: MatrixFrequency,
+        area_id: str | None = None,
+        area_from_id: str | None = None,
```

Comment:
I notice that we are not completely consistent between use of `area1` and  `area_from`, but in the API it seems that we use more `area_from` which is consistent with this